Completion for plugin goals and configuration not working on Windows

When defining a plugin execution configuration in the new lemminx editor, support completion of the plugin goals.

```
					<plugin>
						<groupId>org.apache.maven.plugins</groupId>
						<artifactId>maven-dependency-plugin</artifactId>
						<version>3.0.0</version>
						<executions>
							<execution>
								<id>download-specs</id>
								<phase>package</phase>
								<goals>
									<goal></goal>
								</goals>
							</execution>
						</executions>
					</plugin>
```
Pressing CTRL+Space at goals above should give me the list of plugin goals supported by the dependency-plugin.

```
[ERROR] Failures:
[ERROR]   LocalPluginTest.testCompleteConfigurationParameters:61
[ERROR]   LocalPluginTest.testCompleteConfigurationParametersInTag:68
[ERROR]   LocalPluginTest.testCompleteGoal:54
[ERROR]   LocalPluginTest.testPluginConfigurationDiagnostics:98
[ERROR]   LocalPluginTest.testPluginGoalDiagnostics:106
[ERROR]   LocalRepoTests.testCompleteDependency:45
[ERROR]   LocalRepoTests.testCompleteLocalGroupdId:52
[ERROR]   SimpleModelTest.testCompletePhase:126
[ERROR]   SimpleModelTest.testCompleteScope:117
[ERROR]   SimpleModelTest.testMissingArtifactIdError:104
[ERROR]   SimpleModelTest.testParentPropertyCompletion:82
[ERROR]   SimpleModelTest.testPropertyCompletion:74
[ERROR]   SimpleModelTest.testScopeCompletion:65
[ERROR] Errors:
[ERROR]   LocalPluginTest.testPluginArtifactHover:90 NullPointer
[ERROR]   LocalPluginTest.testPluginConfigurationHover:78 NullPointer
[ERROR]   LocalPluginTest.testPluginGoalHover:84 NullPointer
[ERROR]   RemoteRepositoryTest.testRemoteArtifactHover:88 NullPointer
[ERROR]   RemoteRepositoryTest.testRemoteArtifactIdCompletion:67->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemoteGroupIdCompletion:60->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginArtifactIdCompletion:101->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginGroupIdCompletion:94->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginVersionCompletion:108->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemoteVersionCompletion:74->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   SimpleModelTest.testPropertyHover:136 NullPointer
[INFO]
[ERROR] Tests run: 29, Failures: 13, Errors: 11, Skipped: 0
[
```

```
[ERROR] Tests run: 29, Failures: 13, Errors: 11, Skipped: 0, Time elapsed: 229.594 s <<< FAILURE! - in org.eclipse.lemminx.maven.test.AllTests
[ERROR] testCompleteLocalGroupdId(org.eclipse.lemminx.maven.test.LocalRepoTests)  Time elapsed: 2.102 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalRepoTests.testCompleteLocalGroupdId(LocalRepoTests.java:52)

[ERROR] testCompleteDependency(org.eclipse.lemminx.maven.test.LocalRepoTests)  Time elapsed: 0.139 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalRepoTests.testCompleteDependency(LocalRepoTests.java:45)

[ERROR] testPluginConfigurationDiagnostics(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.138 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginConfigurationDiagnostics(LocalPluginTest.java:98)

[ERROR] testPluginGoalHover(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.083 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginGoalHover(LocalPluginTest.java:84)

[ERROR] testCompleteConfigurationParametersInTag(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.081 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testCompleteConfigurationParametersInTag(LocalPluginTest.java:68)

[ERROR] testPluginGoalDiagnostics(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.072 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginGoalDiagnostics(LocalPluginTest.java:106)

[ERROR] testPluginConfigurationHover(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.08 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginConfigurationHover(LocalPluginTest.java:78)

[ERROR] testCompleteGoal(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.064 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testCompleteGoal(LocalPluginTest.java:54)

[ERROR] testCompleteConfigurationParameters(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.069 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testCompleteConfigurationParameters(LocalPluginTest.java:61)

[ERROR] testPluginArtifactHover(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.068 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginArtifactHover(LocalPluginTest.java:90)

[ERROR] testRemoteVersionCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.011 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemoteVersionCompletion(RemoteRepositoryTest.java:74)

[ERROR] testRemoteGroupIdCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.015 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemoteGroupIdCompletion(RemoteRepositoryTest.java:60)

[ERROR] testRemotePluginVersionCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.019 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemotePluginVersionCompletion(RemoteRepositoryTest.java:108)

[ERROR] testRemotePluginGroupIdCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.007 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemotePluginGroupIdCompletion(RemoteRepositoryTest.java:94)

[ERROR] testRemoteArtifactHover(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 0.568 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemoteArtifactHover(RemoteRepositoryTest.java:88)

[ERROR] testRemotePluginArtifactIdCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.008 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemotePluginArtifactIdCompletion(RemoteRepositoryTest.java:101)

[ERROR] testRemoteArtifactIdCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 150.005 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 150000 milliseconds
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemoteArtifactIdCompletion(RemoteRepositoryTest.java:67)

[ERROR] testScopeCompletion(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.075 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testScopeCompletion(SimpleModelTest.java:65)

[ERROR] testMissingArtifactIdError(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.063 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testMissingArtifactIdError(SimpleModelTest.java:104)

[ERROR] testParentPropertyCompletion(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.069 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testParentPropertyCompletion(SimpleModelTest.java:82)

[ERROR] testPropertyCompletion(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.068 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testPropertyCompletion(SimpleModelTest.java:74)

[ERROR] testPropertyHover(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.047 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testPropertyHover(SimpleModelTest.java:136)

[ERROR] testCompletePhase(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.064 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testCompletePhase(SimpleModelTest.java:126)

[ERROR] testCompleteScope(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.074 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testCompleteScope(SimpleModelTest.java:117)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   LocalPluginTest.testCompleteConfigurationParameters:61
[ERROR]   LocalPluginTest.testCompleteConfigurationParametersInTag:68
[ERROR]   LocalPluginTest.testCompleteGoal:54
[ERROR]   LocalPluginTest.testPluginConfigurationDiagnostics:98
[ERROR]   LocalPluginTest.testPluginGoalDiagnostics:106
[ERROR]   LocalRepoTests.testCompleteDependency:45
[ERROR]   LocalRepoTests.testCompleteLocalGroupdId:52
[ERROR]   SimpleModelTest.testCompletePhase:126
[ERROR]   SimpleModelTest.testCompleteScope:117
[ERROR]   SimpleModelTest.testMissingArtifactIdError:104
[ERROR]   SimpleModelTest.testParentPropertyCompletion:82
[ERROR]   SimpleModelTest.testPropertyCompletion:74
[ERROR]   SimpleModelTest.testScopeCompletion:65
[ERROR] Errors:
[ERROR]   LocalPluginTest.testPluginArtifactHover:90 NullPointer
[ERROR]   LocalPluginTest.testPluginConfigurationHover:78 NullPointer
[ERROR]   LocalPluginTest.testPluginGoalHover:84 NullPointer
[ERROR]   RemoteRepositoryTest.testRemoteArtifactHover:88 NullPointer
[ERROR]   RemoteRepositoryTest.testRemoteArtifactIdCompletion:67->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemoteGroupIdCompletion:60->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginArtifactIdCompletion:101->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginGroupIdCompletion:94->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginVersionCompletion:108->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemoteVersionCompletion:74->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   SimpleModelTest.testPropertyHover:136 NullPointer
[INFO]
[ERROR] Tests run: 29, Failures: 13, Errors: 11, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:56 min
[INFO] Finished at: 2020-05-13T14:11:41+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project lemminx-maven: There are test failures.
[ERROR]
[ERROR] Please refer to C:\eclipse_installations\testing\lemminx-maven\lemminx-maven\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

```
still same without any eclipse running while this was executed

```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
May 13, 2020 8:26:30 PM org.eclipse.lemminx.uriresolver.CacheResourcesManager lambda$downloadResource$0
INFO: Downloading http://maven.apache.org/xsd/maven-4.0.0.xsd to C:\Users\gayan\.lemminx\cache\http\maven.apache.org\xsd\maven-4.0.0.xsd...
May 13, 2020 8:26:30 PM org.eclipse.lemminx.uriresolver.CacheResourcesManager lambda$downloadResource$0
INFO: Downloaded http://maven.apache.org/xsd/maven-4.0.0.xsd to C:\Users\gayan\.lemminx\cache\http\maven.apache.org\xsd\maven-4.0.0.xsd in 422ms
May 13, 2020 8:30:21 PM org.eclipse.lemminx.uriresolver.CacheResourcesManager lambda$downloadResource$0
INFO: Downloading https://maven.apache.org/xsd/maven-4.0.0.xsd to C:\Users\gayan\.lemminx\cache\https\maven.apache.org\xsd\maven-4.0.0.xsd...
May 13, 2020 8:30:22 PM org.eclipse.lemminx.uriresolver.CacheResourcesManager lambda$downloadResource$0
INFO: Downloaded https://maven.apache.org/xsd/maven-4.0.0.xsd to C:\Users\gayan\.lemminx\cache\https\maven.apache.org\xsd\maven-4.0.0.xsd in 968ms
[ERROR] Tests run: 29, Failures: 13, Errors: 11, Skipped: 0, Time elapsed: 236.42 s <<< FAILURE! - in org.eclipse.lemminx.maven.test.AllTests
[ERROR] testCompleteLocalGroupdId(org.eclipse.lemminx.maven.test.LocalRepoTests)  Time elapsed: 3.256 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalRepoTests.testCompleteLocalGroupdId(LocalRepoTests.java:52)

[ERROR] testCompleteDependency(org.eclipse.lemminx.maven.test.LocalRepoTests)  Time elapsed: 1.557 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalRepoTests.testCompleteDependency(LocalRepoTests.java:45)

[ERROR] testPluginConfigurationDiagnostics(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.323 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginConfigurationDiagnostics(LocalPluginTest.java:98)

[ERROR] testPluginGoalHover(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.213 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginGoalHover(LocalPluginTest.java:84)

[ERROR] testCompleteConfigurationParametersInTag(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.25 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testCompleteConfigurationParametersInTag(LocalPluginTest.java:68)

[ERROR] testPluginGoalDiagnostics(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.203 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginGoalDiagnostics(LocalPluginTest.java:106)

[ERROR] testPluginConfigurationHover(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.266 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginConfigurationHover(LocalPluginTest.java:78)

[ERROR] testCompleteGoal(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.188 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testCompleteGoal(LocalPluginTest.java:54)

[ERROR] testCompleteConfigurationParameters(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.203 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testCompleteConfigurationParameters(LocalPluginTest.java:61)

[ERROR] testPluginArtifactHover(org.eclipse.lemminx.maven.test.LocalPluginTest)  Time elapsed: 0.203 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.LocalPluginTest.testPluginArtifactHover(LocalPluginTest.java:90)

[ERROR] testRemoteVersionCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.015 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemoteVersionCompletion(RemoteRepositoryTest.java:74)

[ERROR] testRemoteGroupIdCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.009 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemoteGroupIdCompletion(RemoteRepositoryTest.java:60)

[ERROR] testRemotePluginVersionCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.011 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemotePluginVersionCompletion(RemoteRepositoryTest.java:108)

[ERROR] testRemotePluginGroupIdCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 15.017 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemotePluginGroupIdCompletion(RemoteRepositoryTest.java:94)

[ERROR] testRemoteArtifactHover(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 0.901 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemoteArtifactHover(RemoteRepositoryTest.java:88)

[ERROR] testRemotePluginArtifactIdCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 14.995 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 15000 milliseconds
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemotePluginArtifactIdCompletion(RemoteRepositoryTest.java:101)

[ERROR] testRemoteArtifactIdCompletion(org.eclipse.lemminx.maven.test.RemoteRepositoryTest)  Time elapsed: 150.012 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 150000 milliseconds
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.loopUntilCompletionItemFound(RemoteRepositoryTest.java:54)
        at app//org.eclipse.lemminx.maven.test.RemoteRepositoryTest.testRemoteArtifactIdCompletion(RemoteRepositoryTest.java:67)

[ERROR] testScopeCompletion(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.248 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testScopeCompletion(SimpleModelTest.java:65)

[ERROR] testMissingArtifactIdError(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.195 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testMissingArtifactIdError(SimpleModelTest.java:104)

[ERROR] testParentPropertyCompletion(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.21 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testParentPropertyCompletion(SimpleModelTest.java:82)

[ERROR] testPropertyCompletion(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testPropertyCompletion(SimpleModelTest.java:74)

[ERROR] testPropertyHover(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.438 s  <<< ERROR!
java.lang.NullPointerException
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testPropertyHover(SimpleModelTest.java:136)

[ERROR] testCompletePhase(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.268 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testCompletePhase(SimpleModelTest.java:126)

[ERROR] testCompleteScope(org.eclipse.lemminx.maven.test.SimpleModelTest)  Time elapsed: 0.228 s  <<< FAILURE!
java.lang.AssertionError
        at org.eclipse.lemminx.maven.test.SimpleModelTest.testCompleteScope(SimpleModelTest.java:117)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   LocalPluginTest.testCompleteConfigurationParameters:61
[ERROR]   LocalPluginTest.testCompleteConfigurationParametersInTag:68
[ERROR]   LocalPluginTest.testCompleteGoal:54
[ERROR]   LocalPluginTest.testPluginConfigurationDiagnostics:98
[ERROR]   LocalPluginTest.testPluginGoalDiagnostics:106
[ERROR]   LocalRepoTests.testCompleteDependency:45
[ERROR]   LocalRepoTests.testCompleteLocalGroupdId:52
[ERROR]   SimpleModelTest.testCompletePhase:126
[ERROR]   SimpleModelTest.testCompleteScope:117
[ERROR]   SimpleModelTest.testMissingArtifactIdError:104
[ERROR]   SimpleModelTest.testParentPropertyCompletion:82
[ERROR]   SimpleModelTest.testPropertyCompletion:74
[ERROR]   SimpleModelTest.testScopeCompletion:65
[ERROR] Errors:
[ERROR]   LocalPluginTest.testPluginArtifactHover:90 NullPointer
[ERROR]   LocalPluginTest.testPluginConfigurationHover:78 NullPointer
[ERROR]   LocalPluginTest.testPluginGoalHover:84 NullPointer
[ERROR]   RemoteRepositoryTest.testRemoteArtifactHover:88 NullPointer
[ERROR]   RemoteRepositoryTest.testRemoteArtifactIdCompletion:67->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemoteGroupIdCompletion:60->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginArtifactIdCompletion:101->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginGroupIdCompletion:94->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemotePluginVersionCompletion:108->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   RemoteRepositoryTest.testRemoteVersionCompletion:74->loopUntilCompletionItemFound:54 » TestTimedOut
[ERROR]   SimpleModelTest.testPropertyHover:136 NullPointer
[INFO]
[ERROR] Tests run: 29, Failures: 13, Errors: 11, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  08:37 min
[INFO] Finished at: 2020-05-13T20:30:29+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project lemminx-maven: There are test failures.
[ERROR]
[ERROR] Please refer to C:\Users\gayan\projects\lemminx-maven\lemminx-maven\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].
```
This is from fresh installed windows virtual box.

--- a/lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPlugin.java
+++ b/lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPlugin.java
@@ -12,6 +12,7 @@
 package org.eclipse.lemminx.maven;
 
 import java.io.File;
+import java.net.URI;
 
 import org.apache.maven.Maven;
 import org.apache.maven.artifact.repository.ArtifactRepository;
@@ -233,9 +234,9 @@ public class MavenPlugin implements IXMLExtension {
 	}
 
 	public static boolean match(DOMDocument document) {
-		String uri = document.getDocumentURI();
-		String fileName = uri.substring(uri.lastIndexOf(File.separatorChar) + 1);
-		return (fileName.startsWith("pom") && fileName.endsWith(".xml")) || fileName.endsWith(Maven.POMv4);
+		File file = new File(URI.create(document.getDocumentURI()));
+		return (file.getName().startsWith("pom") && file.getName().endsWith(".xml"))
+				|| file.getName().endsWith(Maven.POMv4);
 	}
 
 	public MavenProjectCache getProjectCache() {

